db/projects/monetha/asset-mth-logo-square.svg

```diff
@@ -0,0 +1 @@
+<svg height="32" viewBox="0 0 32 32" width="32" xmlns="http://www.w3.org/2000/svg"><g fill="none"><circle cx="16" cy="16" fill="#104fca" r="16"/><g fill="#fff" transform="translate(7 9)"><path d="m0 .56757699 4.2345 7.10328525v7.32913776c-2.33865 0-4.2345-1.8360837-4.2345-4.101075z" fill-opacity=".5"/><path d="m13.76487 7.66320318 4.2319086-7.09893822h.0025914v10.33466004c0 2.2649913-1.89585 4.101075-4.2345 4.101075z" fill-opacity=".6"/><path d="m8.99739174 7.45835186-2.11725174 3.55161324c-.43914-.1106914-.91125-.5363451-1.41633-1.27696103l-5.46381-9.16542708c2.025-1.13245206 4.61511-.46037188 5.78439 1.50113306z" fill-opacity=".8"/><path d="m12.21561 2.05990703c1.16928-1.96150494 3.75939-2.63358512 5.78439-1.5010459l-5.46381 9.16542707c-.75762 1.1109239-2.0583 1.843928-3.53583 1.843928-.65968947.0009185-1.3103983-.1480775-1.89981-.4350088l-.22041-.1232423z"/></g></g></svg>
```

Comment:
Maybe the full logo is more like this one? asset-mth-logo-full.svg
https://www.monetha.io/3a89703038409d26fd9db1370501dfba.svg

The one you are mentioning here is based on my opinion asset-mth-logo-alternative.svg

Reply:
switched alternative to square